pytube/contrib/channel.py

```diff
@@ -199,3 +199,99 @@ def _extract_videos(raw_json: str) -> Tuple[List[str], Optional[str]]:
             ),
             continuation,
         )
+
+    @staticmethod
+    def _extract_playlists(raw_json: str) -> Tuple[List[str], Optional[str]]:
+        """Extracts playlists from a raw json page
+
+        :param str raw_json: Input json extracted from the page or the last
+            server response
+        :rtype: Tuple[List[str], Optional[str]]
+        :returns: Tuple containing a list of up to 100 video watch ids and
+            a continuation token, if more videos are available
+        """
+        initial_data = json.loads(raw_json)
+
+        # this is the json tree structure, if the json was extracted from
+        # html
+        try:
+            playlists = initial_data["contents"][
+                "twoColumnBrowseResultsRenderer"][
+                "tabs"][2]["tabRenderer"]["content"][
+                "sectionListRenderer"]["contents"][0][
+                "itemSectionRenderer"]["contents"][0][
+                'shelfRenderer']["content"][
+                'horizontalListRenderer']["items"]
+        except (KeyError, IndexError, TypeError):
+            playlists = []
+
+        continuation = None
+        # remove duplicates
+        return (
+            uniqueify(
+                list(
+                    # only extract the video ids from the video data
+                    map(
+                        lambda x: (
+                            f"/playlist?list="
+                            f"{x['gridPlaylistRenderer']['playlistId']}"
+                        ),
+                        playlists
+                    )
+                ),
+            ),
+            continuation,
```

Comment:
continuation is never populated?

Reply:
We don't know how to extract the continuation information currently, because we'd need to find an example channel with something like >50 playlists created on it (which is extremely unlikely to be the case). In the future, when somebody discovers an example of this, we can fill in the continuation code as necessary.

Reply:
@tfdahlin  Comedy Central's should have ~ 140+ :-) https://www.youtube.com/c/ComedyCentral/playlists